packages/synapse-react-client/src/style/base/_icons.scss

```diff
@@ -81,7 +81,7 @@
 svg.HelpButton {
   color: map.get(SRC.$colors, 'gray-600');
   cursor: pointer;
-  font-size: 0.8em;
+  font-size: 1.5em;
```

Comment:
Small fix for HelpButton size, which was way too small.
Before:
<img width="979" alt="Screen Shot 2024-05-08 at 10 51 43 AM" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/1864447/2df994a3-ba16-4f2c-8b64-3d403a67279a">

After:
<img width="976" alt="Screen Shot 2024-05-08 at 10 51 54 AM" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/1864447/0d15cde2-0d05-4ff5-827e-0b102b70c34e">